<launch>

    <node pkg="swanson_sensors" type="rtimu_ros_node" name="imu_node" output="screen">
         <param name="root_path" value="$(find swanson_sensors)/config"/>
         <param name="config_file" value="razor_m0_imu"/>
         <param name="imu_topic" value="imu/data/raw"/>
         <param name="mag_topic" value="imu/mag"/>
         <param name="pose_topic" value="imu/pose/raw"/>
    </node>

</launch>
